assets/js/state/sagas/softwareUpdatesSettings.js

```diff
@@ -61,6 +63,19 @@ export function* updateSoftwareUpdatesSettings(payload) {
   }
 }
 
+export function* clearSoftwareUpdatesSettings() {
+  yield put(startLoadingSoftwareUpdatesSettings());
+
+  try {
+    yield call(clearSettings);
+
+    yield put(setEmptySoftwareUpdatesSettings());
+  } catch (error) {
+    const errors = get(error, ['response', 'data', 'errors'], []);
+    yield put(setSoftwareUpdatesSettingsErrors(errors));
```

Comment:
I think there's no need to set errors, this is meant for validation errors in other sagas